### ISSUE TITLE
How to configure the precision of the p-value scores?

### ISSUE THREAD

**Issue by andmek** [Unbabel/COMET#81]
## How to configure the precision of the p-value scores?

_comet-compare_ normally displays the p-value scores with four decimal digits (e.g., p-value: 0.6059). In my evaluation, some of the p-values have zero scores (p-value: 0.0000). I want to have p-values with more than four decimal digits. Is there an option to do that?

**Comment by ricardorei** [comment]
Hi @andmek! Not optionally no.. but that's something easy to change. You just have to change [this function ](https://github.com/Unbabel/COMET/blob/master/comet/cli/compare.py#L77) and replace all `print("{}:\t{:.4f}".format(k, v))` with `print("{}:\t{}".format(k, v))` for example.

**Comment by andmek** [comment]
Thanks, @ricardorei! I made the precision 80 decimal points! Still, I've got a result like this:

Bootstrap Resampling Results:
x-mean:	-0.26859185950079933480338922890950925648212432861328125000000000000000000000000000
y-mean:	0.28345951165515925751492432027589529752731323242187500000000000000000000000000000
ties (%):	0.00000000000000000000000000000000000000000000000000000000000000000000000000000000
x_wins (%):	0.00000000000000000000000000000000000000000000000000000000000000000000000000000000
y_wins (%):	1.00000000000000000000000000000000000000000000000000000000000000000000000000000000

Paired T-Test Results:
statistic:	-54.90133764783409020537874312140047550201416015625000000000000000000000000000000000
p_value:	0.00000000000000000000000000000000000000000000000000000000000000000000000000000000
Null hypothesis rejected according to t-test.

Is it alright?
